src/Core/Services/MetadataProviders/MySqlMetadataProvider.cs

```diff
@@ -128,12 +129,124 @@ protected override DatabaseTable GenerateDbTable(string schemaName, string table
         }
 
         /// <summary>
-        /// Takes a string version of a MySql data type and returns its .NET common language runtime (CLR) counterpart
-        /// TODO: For MySql stored procedure support, this needs to be implemented.
+        /// Takes a string version of a PostgreSql data type and returns its .NET common language runtime (CLR) counterpart
```

Comment:
If there are changes related to MySQL as well, can you move them in a separate PR?

Since, the PR title says it's for Postgres